app/views/feedback/_form.html.haml

```diff
@@ -1,6 +1,9 @@
-= GovukElementsErrorsHelper.error_summary @feedback, "There is a problem with the form", ""
 - if flash[:success].present?
   .alert--success= flash[:success]
+- if flash[:alert].present?
+  - @feedback.valid?
```

Comment:
you need to run this check to get the errors to populate, this is taken from https://govuk-elements-rails-guide.herokuapp.com/error_validations#form-error-validation

Reply:
Oh ok. I think you should be ok without it because it's already called in the controller. That's normally where I'd expect any side effects to happen.